catchpoint/scripts/init_script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+sudo snap install kubectl --classic
+
+# Determine the architecture
+ARCH=$(uname -m)
+if [ "$ARCH" = "x86_64" ]; then
+    MINIKUBE_BIN="minikube-linux-amd64"
+else
+    MINIKUBE_BIN="minikube-linux-arm64"
+fi
+
+# Setup Minikube
+curl -LO "https://storage.googleapis.com/minikube/releases/latest/$MINIKUBE_BIN"
+sudo install $MINIKUBE_BIN /usr/local/bin/minikube
+minikube start --memory=12000 --cpus=4 --kubernetes-version=v1.23.17 # Version for Docker support
+GATEWAY_IP="$(ip r | grep default | cut -d' ' -f3)"
+echo "$GATEWAY_IP catchpoint.k3d.localhost" | sudo tee -a /etc/hosts
+
+sudo snap install helm --classic
+
+# Apply Kubernetes resources for Catchpoint Exporter from the generated_configs directory
+kubectl apply -f /catchpoint/config/catchpoint-exporter-deployment.yaml
+kubectl apply -f /catchpoint/config/catchpoint-exporter-service.yaml
+
+# Install Grafana Agent Flow
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm install grafana-agent-flow grafana/k8s-monitoring -n default --values /catchpoint/config/grafana-flow-k8s-template.yaml
```

Comment:
I'm getting this issue, is this anything I'm missing or misconfigured on my end?
```
Update Complete. ⎈Happy Helming!⎈
W0529 17:44:42.801404    5988 warnings.go:70] policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
Error: INSTALLATION FAILED: failed pre-install: warning: Hook pre-install k8s-monitoring/templates/hooks/validate-configuration.yaml failed: 1 error occurred:
        * pods "validate-grafana-agent-flow-k8s-monitoring" is forbidden: error looking up service account default/default: serviceaccount "default" not found

```

Reply:
It may be due to invalid credentials. I updated to include a new prom host value which has the region. You can try again with the prom_host, prom_user and prom_pass, loki_user and loki_pass filled out and see if that works.

Reply:
@Dasomeone I've tested this out as well, and can confirm it works